website/package.json

```diff
@@ -30,6 +30,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-slick": "^0.30.2",
+    "serve": "^14.2.3",
```

Comment:
Do we need this?

Reply:
I am unsure why but let me remove this line